README.md

```diff
@@ -41,13 +112,84 @@ terraform plan -out=test-plan.tfplan
 Apply infrastructure
 
 ```bash
-terraform apply
+terraform apply -var infrastructure_db_password=<db-password> -var aap_admin_password=<aap-admin-password> -var aap_red_hat_username=<redhat-username> -var aap_red_hat_password=<redhat-password> -var infrastructure_controller_ami="" -var infrastructure_hub_ami="" -var infrastructure_execution_ami="" -var infrastructure_eda_ami=""
```

Comment:
If we leave these blank values for the *_ami in the example, we should add a note as to why, because passing blank values makes it look like it needs to be changed by the user.

Reply:
I am removing it from here. and passign default = "" in variables. Also updatign Readme to tell about this